deploy/spoke/clusterrole.yaml

```diff
@@ -3,20 +3,12 @@ kind: ClusterRole
 metadata:
   name: open-cluster-management:spoke
 rules:
+# Allow agent to get/list/watch nodes
+# list nodes to calculates the capacity and allocatable resources of the managed cluster
 - apiGroups: [""]
-  resources: ["nodes", "configmaps"]
+  resources: ["nodes"]
   verbs: ["get", "list", "watch"]
-- apiGroups: [""]
-  resources: ["secrets"]
-  verbs: ["get", "list", "watch", "create", "delete", "update", "patch"]
-- apiGroups: ["authorization.k8s.io"]
-  resources: ["subjectaccessreviews"]
-  verbs: ["create"]
 # Allow agent to list clusterclaims
 - apiGroups: ["cluster.open-cluster-management.io"]
   resources: ["clusterclaims"]
-  verbs: ["get", "list", "watch"]
-# Allow agent to list addons lease
-- apiGroups: ["coordination.k8s.io"]
-  resources: ["leases"]
-  verbs: ["get", "list", "watch"]
+  verbs: ["get", "list", "watch"]
```

Comment:
please add an empty line in the end of file

Reply:
fixed